src/script/media/MediaDevicesHandler.ts

```diff
@@ -163,14 +169,22 @@ export class MediaDevicesHandler {
 
   /**
    * Initialize the list of MediaDevices and subscriptions.
+   * @camera: boolean, Only when the camera is queried can the entire device list be accessed.
+   * @return Promise<void>
    */
-  private initializeMediaDevices(): void {
-    if (Runtime.isSupportingUserMedia()) {
-      this.refreshMediaDevices().then(() => {
+  public async initializeMediaDevices(camera: boolean = false): Promise<void> {
```

Comment:
You don't have to assign type there, because default value assign type to arg ;)

```suggestion
  public async initializeMediaDevices(camera = false): Promise<void> {
```